python/pyspark/sql/session.py

```diff
@@ -393,6 +393,8 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
         >>> l = [('Alice', 1)]
         >>> spark.createDataFrame(l).collect()
         [Row(_1=u'Alice', _2=1)]
+        >>> spark.createDataFrame(l, ['name', u'age']).columns
```

Comment:
Yes, these will become part of the API docs as examples, it's anonying to see many corner cases here.